### PR TITLE
Added confirmation page content to schema

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Added ability to load confirmation page content from schema

### How to review 
Check that the confirmation page content is displaying what is defined in the schema

### CODE REVIEW

**Review by benjefferies — APPROVED** [review]
Although the structure is right we should probably get someone to double check it looks close enough.

**Comment by yoganandkunche** [comment]
Content on the final confirmation page match's the spec for ***Census_Communal***

Title on the confirmation page does not match for ***Census_Individual***

Expected: You’re ready to submit your 2017 Census Test
Actual: You’ve completed the 2017 Census Test

Title on the confirmation page does not match for ***Census Household*** 

Expected: You’re ready to submit your 2017 Census Test
Actual: You’ve completed the 2017 Census Test

Also 
Content with in bullet points did not match. Please find attached screenshot
screen shot 2016-12-10 at 15 17 12

screen shot 2016-12-10 at 15 11 35

**Comment by jonnyshaw89** [comment]
@yoganandkunche i have updated the page titles. my version of the slides doesn't look the same as what you have the screenshot of. can you confirm you have the latest version?

**Comment by yoganandkunche** [comment]
@jonnyshaw89 There is a bit of confusion with the version at my end.. now gone through the latest version and content looks good. Then the only issue is the title on Census_Individual and Census_household which you fixed. I will be retesting them now.

File: app/templates/confirmation.html [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
+{% extends theme('layouts/_survey.html') %}
 
-{% block page_title -%}Summary{% endblock -%}
+{% block page_title %}Confirmation{% endblock %}
 
-{% block main -%}
+{% block main %}
 
-<a class="u-dib" href="{{previous_location}}">Previous</a>
+<div class="grid">
```
Comment (hamishtaplin) [inline app/templates/confirmation.html]: This needs to extend an existing template an override where necessary rather than duplicate one.

**Comment by yoganandkunche** [comment]
Not merged into master yet. So will be testing it in pre-prod tomorrow.

**Review by bennpearson — APPROVED** [review]

**Comment by jonnyshaw89** [comment]
👍

**Pull request merged**